### ISSUE TITLE
[Tab Extended]: QA: Tooltip of tab item is not dismissible with esc key

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9232]
### Description

Page Element: Tooltip content

Issue Description:
User is unable to dismiss the tooltip content using esc key.

Expected:User should be able to dismiss the tooltip using esc key.

User Impact:
When the user is unable to dismiss the tooltip using esc key, motor impairment users will not be able to access the content.

### Component(s) impacted

Tab extended
Tab extended with media

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component canary: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
